gitleaves/templates/ghwiki/Monthwise.md

```diff
@@ -0,0 +1,14 @@
+{% for month, people in people_by_month.items() %}
+
+## {{ month }}  
+
+{% for name, days in people.items() %}  
+**{{ name }} :**  
+
+{%- for day in days %}  
+{{ day.strftime('%Y-%m-%d') }}  
```

Comment:
How about `### {{ name }}` vs `**{{ name }}**`

Also Ordered list might be better for listing days

`1. {{ day.strftime('%Y-%m-%d') }} `

Reply:
- Did not think of name been a structurally different thing from the dates. Makes sense! Changed to h3
- I thought about doing a list, then second guessed myself :) Dates are now lists.